Dockerfile

```diff
@@ -7,10 +7,50 @@ ENV PIP_INDEX_URL="https://pypi.pacificclimate.org/simple/"
 ENV THREDDS_URL_ROOT="https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/thredds/dodsC/datasets"
 
 # Update system
-RUN apk add \
+RUN apk upgrade --update && \
+  apk add \
   libxml2-dev \
   libxslt-dev \
-  linux-headers
+  linux-headers \
+  # R/Rpy2 dependencies (for wps-tools)
+  # based on https://hub.docker.com/r/achillesrasquinha/rpy2/dockerfile
+  gcc \
+  gfortran \
+  g++ \
+  make \
+  readline-dev \
+  icu-dev \
+  bzip2-dev \
+  xz-dev \
+  pcre-dev \
+  libjpeg-turbo-dev \
+  libpng-dev \
+  tiff-dev  \
+  curl-dev \
+  zip \
+  file \
+  coreutils \
+  musl-dev \
+  libffi-dev \
+  python3-dev \
+  py3-pip \
+  bash
```

Comment:
This feels like a lot of requirements. Are they all necessary?

Reply:
I agree, I can see if I can find some other option. I tried only having `gcc` and `libffi-dev` just because of the errors I was getting, but rpy2 would not install. Every resource I looked at for `R` on `alpine` seems to have a similarly long list of dependencies.
https://hub.docker.com/r/achillesrasquinha/rpy2/dockerfile
https://www.r-bloggers.com/2018/11/building-a-repository-of-alpine-based-docker-images-for-r-part-i/
https://github.com/velaco/alpine-r/blob/master/base-3.5.0-r1/Dockerfile

Reply:
I mean these will just sit in the dockerfile and aren't really an issue. So if we need them all that is fine, it just seems like a lot haha.

Reply:
ya :/ based on what I've read, alpine doesn't support R like other builds (which would only require `apt install r-base` and maybe `r-dev`)